docs/sql-guide/statements/statement-insert.md

```diff
@@ -46,6 +46,9 @@ There are special assignments for certain literal values.
 | `,NULL,`(case insensitive) | All unless explicitly listed | `NULL`| |
 | `[]` | `stringset` <br/>`idset` | `[]` (empty set) | Stores an empty set for new records and existing `NULL` records. Keeps existing values in set otherwise |
 
+## Special cautions
```

Comment:
Something I haven't documented in "Help on Help" is a heading for stuff like this. But I've put it into place here:

https://docs.featurebase.com/docs/community/com-cluster/com-cluster-migrate/

The heading is `## Warnings` because it's straight to the point and it's clearer.

Something else I'd do is structure the warning like this:

```
## Warnings

FeatureBase recommends limiting INSERT statements to one record per key to avoid unexpected results.
Issues may occur under when a single INSERT statement contains multiple records that:
* share the same key
* are being set to NULL and non-NULL values
```